framer/Print.coffee

```diff
@@ -35,7 +35,7 @@ class Printer
 
 			@_closeButton = new Layer
 				parent: @_container
-				html: '<svg><g stroke="#B8B8B8"><path d="M1,1 L8,8"></path><path d="M1,8 L8,1"></path></g></svg>'
+				html: '<svg viewBox="0 0 9 9"><g stroke="#B8B8B8"><path d="M1,1 L8,8"></path><path d="M1,8 L8,1"></path></g></svg>'
```

Comment:
This misplaces the icon in Safari and Chrome:

<img width="590" alt="screen shot 2017-02-21 at 17 09 41" src="https://cloud.githubusercontent.com/assets/248861/23173225/c26ef400-f858-11e6-9afa-4b5e9adbc535.png">